README.md

```diff
@@ -1 +1,366 @@
+
+If you want to use wallet you need the following settings at build time:
+* set the "OP_WALLET_CONNECTION" varibale to true and during the start 
+* set the "OP_WALLET_PATH" where you want to have the settings extracted
+* copy the wallet under the containerize/db_wallet folder with the following name: Wallet_DB.zip
+* update the following connection string according to your database settings
+"jdbc:oracle:thin:@opendays_high?TNS_ADMIN=/opt/wallet" (opendays_high is the database service which we connect, TNS_ADMIN is the location where we extract the wallet.) and set the "OP_DATASOURCE_URL" to that
+* set the database credentials for "OP_DATASOURCE_USERNAME" and "OP_DATASOURCE_PW"
+
+**Containerize folder**
+
+This folder contains helper files for the image build.
+* **extra-jars**: The jar files for creating the connection to an Oracle Autonomous database
+* **startup_scripts**: The startup scripts for application building and hosting.
+
+# How to use the container
+
+To run the image you need to have [Docker installed](https://docs.docker.com/install/).
+
+Once your environemnt is ready you update the settings in the ./Dockerfile. 
+Note that you need to use a RedHat or CentOS image. You can use for example the docker hub image : "centos:centos7"
+
+From the root folder, where the Dockerfile is run the following command.
+
+```
+>docker build -t reservation-system-back-end -f ./Dockerfile .
+```
+Note that the maven minor version number can change. Please check the the available [version number](https://downloads.apache.org/maven/maven-3/) before the build and update it in the Dockerfile.
+For more information check the [Maven build section](#Maven_build)
+
+To run the image you need to forward the port 
+
+```
+>docker run -it -p 8080:8080 --rm --name reservation-back-end reservation-system-back-end
+```
+
+You access your image via http://localhost:8080
+
+# Maven build
+
+Maven 3 is used for compiling the application code. During the docker build process we download maven into the image, install the additional dependency jars listed under the containerize/extra-jars folder. For the applicaiton building we used maven 3.6.3, the minor version can change by time so please check and update the version number accordingly what is available [here](https://downloads.apache.org/maven/maven-3/). 
+
+# Running natively
+
+You need to install JAVA 1.8 and Maven 3.6.3 to build, for running the application JAVA 1.8 is required. The solution works perfectly with open JDK 1.8.
+
+For connectiong to an Oracle 19 database you will need the jar-s located under containerize/extra-jars folder. Before the build you need to install the packages from there to your local maven repository. 
+
+
+For installing the dependencies to local maven repository execute the following commands (Note that "/containerize/extra-jars/" is the location of the jar files):
```

Comment:
dependencies --> Oracle jars, OK. I would point to the Oracle download page. They can be got from the Oracle maven https://blogs.oracle.com/dev2dev/get-oracle-jdbc-drivers-and-ucp-from-oracle-maven-repository-without-ides but is a bit of a pain, you have to be registered, etc...

Reply:
Ok I'll remove them.

Reply:
Done, uploaded the changes + updated the text in the readme as well.